themes/luci-theme-openwrt-2020/htdocs/luci-static/openwrt2020/cascade.css

```diff
@@ -1123,6 +1123,19 @@ textarea {
 	padding: .2em 0;
 }
 
+.cbi-value-description a {
+  color: var(--main-bright-color);
+  text-decoration: none;
+  line-height: inherit;
```

Comment:
I'm not a CSS expert but the inherit seems like a hack. Maybe we can avoid it?

https://developer.mozilla.org/en-US/docs/Web/CSS/inherit